### PR TITLE
F#39 ucu map

### LINKED ISSUE #39: Build map of the office and UCU simulation
Using these instructions https://github.com/ucuapps/robert#building-the-map please create maps of the office and ucu world simulation.
Maps should be stored here https://github.com/ucuapps/robert/tree/kinetic-devel/robert_gazebo in the folder maps

### PR DESCRIPTION
**Pull request by deathtanks**
Closes #39 
Created map

### CODE REVIEW

File: robert_gazebo/worlds/ucu.world [hunk 1]
```diff
@@ -9165,6 +9165,410 @@
       <static>1</static>
       <allow_auto_disable>1</allow_auto_disable>
       <pose frame=''>-1.2423 -1.3637 0 0 -0 0</pose>
+    </model>
```
Comment (LyubomyrD) [inline robert_gazebo/worlds/ucu.world]: Why did You changed the ucu.world file?

**Review by LyubomyrD — CHANGES_REQUESTED** [review]
@deathtanks , please, resolve the merge conflict in this branch befor merging

**Comment by LyubomyrD** [comment]
The map seems to be well done

File: robert_gazebo/worlds/ucu.world (hunk 1 above)
Comment (sofiia-tesliuk) [reply robert_gazebo/worlds/ucu.world]: added columns in lectorium

Comment (LyubomyrD) [reply robert_gazebo/worlds/ucu.world]: The columnt were already added here https://github.com/ucuapps/robert/pull/54 weren't they?

**Review by LyubomyrD — APPROVED** [review]

File: robert_gazebo/worlds/ucu.world (hunk 1 above)
Comment (sofiia-tesliuk) [reply robert_gazebo/worlds/ucu.world]: I add them to this branch before #54 was approved.

**Pull request merged**